src/encoded/types/antibody_lot.py

```diff
@@ -158,151 +162,166 @@ def lot_reviews(characterizations, targets, request):
 
-    if not primary_chars:
-        base_review['detail'] = 'Awaiting submission of primary characterization(s) and a ' + \
-            'compliant secondary characterization.'
-        if secondary_status == 'not submitted for review by lab':
-            base_review['status'] = 'not pursued'
-        elif secondary_status in ['compliant', 'exempt from standards']:
-            base_review['status'] = 'partially characterized'
-            base_review['detail'] = 'Awaiting submission of primary characterization(s).'
-        elif secondary_status == 'not compliant':
-            base_review['status'] = 'not characterized to standards'
         else:
-            # Only no secondary or secondary_status in pending dcc review, in progress,
-            # not reviewed or deleted should be left. The status should already be
-            # awaiting characterization by default.
-            pass
-        return [base_review]
-    else:
-        char_reviews = {}
-        for primary in primary_chars:
-            if primary['status'] in ['not reviewed',
-                                     'not submitted for review by lab',
-                                     'in progress']:
-                # Deal with these first as they won't have characterization_reviews
-                if secondary_status == 'not compliant':
-                    # If secondary is not compliant, the status is always not characterized to standards,
-                    # no matter what the primary is.
-                    base_review['status'] = 'not characterized to standards'
-                elif primary['status'] == 'not submitted for review by lab' or \
-                        secondary_status == 'not submitted for review by lab':
-                    base_review['status'] = 'not pursued'
-                elif secondary_status in ['compliant', 'exempt from standards']:
-                    base_review['status'] = 'partially characterized'
-                    base_review['detail'] = 'Awaiting one or more compliant primary characterization(s).'
-                else:
-                    # Only in progress, not reviewed, pending dcc review secondaries should be left
-                    base_review['status'] = 'awaiting characterization'
-
+            # This primary characterization has characterization_reviews
+            for lane_review in primary.get('characterization_reviews', []):
+                # Get the organism information from the lane, not from the target since
+                # there are lanes. Build a new base_review using the lane information
+                base_review = dict()
+                lane_organism = lane_review['organism']
+
+                base_review['biosample_term_name'] = 'any cell type or tissue' \
+                    if is_histone_mod else lane_review['biosample_term_name']
+                base_review['biosample_term_id'] = 'NTR:99999999' \
+                    if is_histone_mod else lane_review['biosample_term_id']
```

Comment:
it seems like for histones you could just ignore the biosample?  I guess it's expected to be non-null.